README.md

```diff
@@ -215,7 +215,29 @@ would generate a template with 5 instances declared.
 
 Finally, the -r option gives you the opportunity to execute some
 arbitrary ruby code in the evaluation context before the cloudformation
-template is evaluated. 
+template is evaluated.
 
+### Rake task
+Simply add the following to your `Rakefile`:
 
+```ruby
+require 'cfndsl/rake_task'
+
+CfnDsl::RakeTask.new do |t|
+  t.cfndsl_opts = {
+    files: [{
+      filename: 'templates/application.rb',
+      output: 'application.json'
+    }],
+    extras: [{
+      yaml: 'templates/default_params.yml'
+    }]
+  }
+end
+```
```

Comment:
I think we need a quick explanation that `output` is optional and how you can use the `verbose` key.